CloudM.Exercise.WebApi/Controllers/PackageController.cs

```diff
@@ -17,6 +19,8 @@ public PackageController()
         [Route("{name}/{version}")]
         public async Task<PackageDependencies> Get(string name, string version)
```

Comment:
It is a good idea to extract the business logic from the controller. Creating a separate service which inject in the controller via dependency injection is a good option.